.travis.yml

```diff
@@ -23,12 +23,12 @@ matrix:
 
 before_install:
   - if [ $TRAVIS_OS_NAME = osx   ]; then brew update ; fi
-  - if [ $TRAVIS_OS_NAME = osx   ]; then brew install python3 ; fi
+  - if [ $TRAVIS_OS_NAME = osx   ]; then brew install python3 || true ; fi
```

Comment:
Does this `true` flag prevent Homebrew from raising an error if python3 is already at installed? That seems to be a recent thing. Previously the Travis osx image had no python3 installed.

Reply:
Yes, it does. The || true is useful in these kinds of situations to ignore an expected error. Works for docker builds too (like adding a user that might have been added already in a base image). I'm not sure why brew returns an error for this condition - it seems that a warning would be enough.